client/app/controllers/collection/add.js

```diff
@@ -109,12 +141,13 @@ export default Ember.Controller.extend({
     },
 
 
-    create_widget_signature: ['widget_component', 'description',
-                                        'section', 'output_parameter', 'action_id'],
+    create_widget_signature: ['widget_component', 'description', 'disabled',
+                                        'css_classes', 'section', 'output_parameter', 'action_id'],
```

Comment:
Create widget can take a couple more paramters. We should look at passing in an object here again.